prime-router/src/main/kotlin/transport/RedoxTransport.kt

```diff
@@ -8,6 +8,8 @@ import com.github.kittinunf.result.Result
 import gov.cdc.prime.router.RedoxTransportType
 import gov.cdc.prime.router.ReportId
 import gov.cdc.prime.router.TransportType
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
 
```

Comment:
Just adding better logging to the transport.

Reply:
OK, bleh:  this may moot the change I just put in my PR (just merged, https://github.com/CDCgov/prime-data-hub/pull/260) in which I pass in the Azure function context into ITransport, to do logging.   Either way, the lack of clarity on what was being sent where was making it very hard to debug, at least on the sftp side.